src/VisualStudio/Core/Def/Implementation/EditAndContinue/VsENCRebuildableProjectImpl.cs

```diff
@@ -1080,11 +860,69 @@ public unsafe int BuildForEnc(object pUpdatePE)
             }
         }
 
+        internal static ENCPROG_ACTIVE_STATEMENT_REMAP[] GetRemapActiveStatements(ImmutableArray<(Guid ThreadId, ActiveInstructionId OldInstructionId, LinePositionSpan NewSpan)> remaps)
+        {
+            var result = new ENCPROG_ACTIVE_STATEMENT_REMAP[remaps.Length];
+            for (int i = 0; i < remaps.Length; i++)
```

Comment:
Consider using `foreach`, or extracting a local for `remaps[i]`.

Reply:
The index is needed. i think it's ok to not extract var.